conjure-java-jaxrs-client/build.gradle

```diff
@@ -23,8 +24,6 @@ dependencies {
         // the shipped version clashes with the newer javax.ws.rs:javax.ws.rs-api used by (e.g.) dropwizard
         exclude group: "javax.ws.rs", module: "jsr311-api"
     }
-    implementation "org.slf4j:slf4j-api"
```

Comment:
I think excavator needs an update to handle this?

Reply:
Considering that out of scope for now, we don't guarantee all slf4j uses are removed, only those that we can cleanly migrate over